src/components/atoms/GpuModeSlider.tsx

```diff
@@ -29,8 +31,8 @@ const GpuModeSlider: FC<{ showSeparator: boolean }> = ({ showSeparator }) => {
   return (
     <>
       <DeckySlider
-        label="GPU Frequency Mode"
-        value={sliderValue || Mode.DEFAULT}
+        label="GPU Mode"
+        value={sliderValue || Mode.BATTERY}
```

Comment:
should probably be `Mode.BALANCE` as the default

Reply:
I thought so too, but if it's set to Mode.BALANCE it will not allow the slider to select battery.  I did test completely removing data and it is setting Mode.BALANCE as default.

Reply:
This should be resolved as well.